### ISSUE TITLE
Can't boot minecraft on M1 Mac with choosing LWJGL v3.3.1

### ISSUE THREAD

**Issue by GoRamen-3292** [PolyMC/PolyMC#1553]
### Operating System
Mac OS 13.2.1 **(arm64)**

### Model of Mac
MacBook Air **M1**, 2020

### Description
First, I launched a Minecraft 1.19.3 instance with various mods in PolyMC v1.4.3. At that time, Minecraft launched properly.
However, as soon as I updated to PolyMC v5.0, the Minecraft 1.19.3 instance with those mods would not start. Just to be sure, I created a vanilla Minecraft 1.19.3 instance and tried to launch it, but it still wouldn't launch.

Log says it ended with exit code 1. There are some LWJGL errors.
Here is the log of the time when I booted vanilla Minecraft 1.19.3.
(Replaced my username with `__user_name__`)

```
PolyMC version: 5.0

Launched instance in online mode

authserver.mojang.com resolves to:
    [13.35.41.62, 13.35.41.43, 13.35.41.41, 13.35.41.49]

session.minecraft.net resolves to:
    [100.24.172.95, 34.197.32.8, 18.214.12.3]

textures.minecraft.net resolves to:
    [2620:1ec:4e:1::46, 2620:1ec:4f:1::46, 13.107.237.46, 13.107.238.46]

api.mojang.com resolves to:
    [13.35.49.109, 13.35.49.30, 13.35.49.22, 13.35.49.87]


Minecraft folder is:
/Users/__user_name__/Library/Application Support/PolyMC/instances/1.19.3/.minecraft


Java path is:
/Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home/bin/java


Checking Java version...
Java is version 17.0.4.1, using 64 (aarch64) architecture, from Azul Systems, Inc..


Main Class:
  net.minecraft.client.main.Main

Native path:
  /Users/__user_name__/Library/Application Support/PolyMC/instances/1.19.3/natives

Traits:
traits FirstThreadOnMacOS
traits XR:Initial

Libraries:
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/lwjgl/lwjgl-openal/3.3.1/lwjgl-openal-3.3.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/ca/weblite/java-objc-bridge/1.1/java-objc-bridge-1.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/github/oshi/oshi-core/6.2.2/oshi-core-6.2.2.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/google/code/gson/gson/2.10/gson-2.10.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/google/guava/guava/31.1-jre/guava-31.1-jre.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/ibm/icu/icu4j/71.1/icu4j-71.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/mojang/authlib/3.16.29/authlib-3.16.29.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/mojang/blocklist/1.0.10/blocklist-1.0.10.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/mojang/brigadier/1.0.18/brigadier-1.0.18.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/mojang/datafixerupper/5.0.28/datafixerupper-5.0.28.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/mojang/javabridge/2.0.25/javabridge-2.0.25.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/mojang/logging/1.1.1/logging-1.1.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/mojang/patchy/2.2.10/patchy-2.2.10.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/commons-codec/commons-codec/1.15/commons-codec-1.15.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/commons-io/commons-io/2.11.0/commons-io-2.11.0.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/commons-logging/commons-logging/1.2/commons-logging-1.2.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/io/netty/netty-buffer/4.1.82.Final/netty-buffer-4.1.82.Final.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/io/netty/netty-codec/4.1.82.Final/netty-codec-4.1.82.Final.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/io/netty/netty-common/4.1.82.Final/netty-common-4.1.82.Final.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/io/netty/netty-handler/4.1.82.Final/netty-handler-4.1.82.Final.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/io/netty/netty-resolver/4.1.82.Final/netty-resolver-4.1.82.Final.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/io/netty/netty-transport-classes-epoll/4.1.82.Final/netty-transport-classes-epoll-4.1.82.Final.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/io/netty/netty-transport-native-unix-common/4.1.82.Final/netty-transport-native-unix-common-4.1.82.Final.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/io/netty/netty-transport/4.1.82.Final/netty-transport-4.1.82.Final.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/it/unimi/dsi/fastutil/8.5.9/fastutil-8.5.9.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/net/java/dev/jna/jna-platform/5.12.1/jna-platform-5.12.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/net/java/dev/jna/jna/5.12.1/jna-5.12.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/net/sf/jopt-simple/jopt-simple/5.0.4/jopt-simple-5.0.4.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/apache/commons/commons-compress/1.21/commons-compress-1.21.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/apache/commons/commons-lang3/3.12.0/commons-lang3-3.12.0.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/apache/httpcomponents/httpcore/4.4.15/httpcore-4.4.15.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/apache/logging/log4j/log4j-api/2.19.0/log4j-api-2.19.0.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/apache/logging/log4j/log4j-core/2.19.0/log4j-core-2.19.0.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/apache/logging/log4j/log4j-slf4j2-impl/2.19.0/log4j-slf4j2-impl-2.19.0.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/joml/joml/1.10.5/joml-1.10.5.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/org/slf4j/slf4j-api/2.0.1/slf4j-api-2.0.1.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/mojang/text2speech/1.11.3/text2speech-1.11.3.jar
  /Users/__user_name__/Library/Application Support/PolyMC/libraries/com/mojang/minecraft/1.19.3/minecraft-1.19.3-client.jar

Native libraries:

Params:
  --username  --version 1.19.3 --gameDir /Users/__user_name__/Library/Application Support/PolyMC/instances/1.19.3/.minecraft --assetsDir /Users/__user_name__/Library/Application Support/PolyMC/assets --assetIndex 2 --uuid  --accessToken  --userType  --versionType release

Window size: 1280 x 720

Java Arguments:
[-XX:+UnlockExperimentalVMOptions, -XX:+UseG1GC, -XX:G1NewSizePercent=20, -XX:G1ReservePercent=20, -XX:MaxGCPauseMillis=30, -XX:G1HeapRegionSize=32M, -Xdock:icon=icon.png, -Xdock:name="PolyMC: 1.19.3", -XstartOnFirstThread, -Xms512m, -Xmx4096m, -Duser.language=en]


Minecraft process ID: 16236


[21:42:18] [main/INFO]: Building optimized datafixer
[21:42:20] [Render thread/INFO]: [STDERR]: [LWJGL] Failed to load a library. Possible solutions:
	a) Add the directory that contains the shared library to -Djava.library.path or -Dorg.lwjgl.librarypath.
	b) Add the JAR that contains the shared library to the classpath.
[21:42:20] [Render thread/INFO]: [STDERR]: [LWJGL] Enable debug mode with -Dorg.lwjgl.util.Debug=true for better diagnostics.
[21:42:20] [Render thread/INFO]: [STDERR]: [LWJGL] Enable the SharedLibraryLoader debug mode with -Dorg.lwjgl.util.DebugLoader=true for better diagnostics.
Feb 16, 2023 9:42:21 PM org.polymc.EntryPoint listen
SEVERE: Exception caught from launcher.
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.polymc.impl.OneSixLauncher.invokeMain(OneSixLauncher.java:104)
	at org.polymc.impl.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:176)
	at org.polymc.impl.OneSixLauncher.launch(OneSixLauncher.java:186)
	at org.polymc.EntryPoint.listen(EntryPoint.java:144)
	at org.polymc.EntryPoint.main(EntryPoint.java:74)
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.mojang.blaze3d.systems.RenderSystem
	at z.a(SourceFile:66)
	at ejf.a(SourceFile:2408)
	at ejf.a(SourceFile:2403)
	at net.minecraft.client.main.Main.a(SourceFile:214)
	at net.minecraft.client.main.Main.main(SourceFile:51)
	... 9 more
Feb 16, 2023 9:42:21 PM org.polymc.EntryPoint main
INFO: Exiting with 1
Minecraftは終了コード 1 で終了しました。
```

Thank you for your support.

**Comment by DisabledMallis** [comment]
Try changing the LWJGL version to the latest. My friend had a similar issue, and using the latest LWJGL fixed the issue, so make sure that is what you're using.

**Comment by GoRamen-3292** [comment]
I see. I'll try to do that later. Thanks!

**Comment by GoRamen-3292** [comment]
I checked LWJGL's version but it was already latest version (v3.3.1) was selected.
But when I selected it to v3.2.2, I was able to start Minecraft.

Here is the log of that time
https://paste.gg/p/anonymous/262aa91c2ae843369bcc0e4c9d3dbc56


![image](https://user-images.githubusercontent.com/106292417/219583481-f7202ef4-7f18-4be8-ba8d-2e88f513e9e4.png)

**Comment by GoRamen-3292** [comment]
The bug was fixed in PolyMC v5.1!
Thanks for fixing!

**Comment by rtsui-harmonicinc** [comment]
same issue observed, also fixed by using 3.2
